assets/stylesheets/components/evolution/paragraph_gallery/README.md

```diff
@@ -0,0 +1,31 @@
+##Paragraph Gallery
+
+The paragraph gallery component allows you to  add contextual images to the end of a paragraph.
+The images should relate to the paragraph topics for maximum impact. Paste the following code to the
+end of your paragraph to include a gallery:
+
+    <div class="paragraph-gallery">
```

Comment:
@imanuelgittens 

Shouldn't this html arkup be wrapped in markdown code block?